### PR TITLE
feat: allow more config values for the Sentry client

### PR DESCRIPTION
**Pull request by sweenu**

### CODE REVIEW

**Comment by sweenu** [comment]
Manually tested it, works fine

File: sentry_wrapper/__init__.py [hunk 1]
```diff
@@ -59,15 +95,42 @@ def execute():
...
+        '--release',
+        nargs='?',
+        help="The program's release version (e.g. 3.2.3)",
+    )
+    parser.add_argument(
+        '--environment',
+        nargs='?',
+        help='The envionment the program is running in (e.g. staging)',
+    )
+    parser.add_argument(
```
Comment (QuentinBrosse) [inline sentry_wrapper/__init__.py:117]: IIRC the `server_name` is automatically deduced from `socket.gethostname()` ([doc](https://docs.sentry.io/clients/python/advanced/)).

In our Go stack (cf. `go-api-tools`) the configuration variable `ServerName` has been deprecated in favor of `ServiceName`. Maybe we should do the same for the Python stack?

File: sentry_wrapper/__init__.py [hunk 2]
```diff
@@ -11,11 +11,47 @@
...
+    def __init__(
+        self,
+        dsn,
+        release=None,
+        environment=None,
+        server_name=None,
+        attach_stacktrace=None,
+        with_locals=None,
+    ):
+        self.dsn = dsn
```
Comment (QuentinBrosse) [inline sentry_wrapper/__init__.py:24]: nit: Do you need to store these variables? Does `self._conf_values` is not enough?

Comment (sweenu) [reply sentry_wrapper/__init__.py:24]: Does not hurt to have them, could be useful later.

File: sentry_wrapper/__init__.py (hunk 1 above)
Comment (sweenu) [reply sentry_wrapper/__init__.py:117]: I just added the options available for the python configuration: https://docs.sentry.io/platforms/python/configuration/options/#server-name.
We don't have to use and probably won't but at least we have the possibility. I don't see any `serivce_name` option in the doc.

**Review by QuentinBrosse — APPROVED** [review]
LGTM

**Pull request merged**